core/components/commerce_timeslots/model/schema/commerce_timeslots.mysql.schema.xml

```diff
@@ -5,14 +5,16 @@
 
     <object class="ctsSchedule" table="commext_timeslots_schedule">
         <field key="name" dbtype="varchar" precision="190" phptype="string" null="false" default="" />
-        <field key="default" dbtype="tinyint" precision="1" phptype="integer" null="false" default="0" />
+        <field key="repeat" dbtype="tinyint" precision="1" phptype="boolean" null="false" default="0" />
+        <field key="repeat_days" dbtype="text" phptype="array" null="false" />
```

Comment:
`null="false"` means it needs a default value to be able of creating records without it on strict database models (or, maybe more appropriate for a serialized field, just set `null="true"` instead)

Reply:
I had thought that xpdo took care of the default when it's phptype array. Hmmm I could definitely be wrong ;)